client/src/nv_ingest_client/cli/util/click.py

```diff
@@ -113,6 +113,10 @@ def click_validate_task(ctx, param, value):
                 task_options = check_schema(StoreTaskSchema, options, task_id, json_options)
                 new_task_id = f"{task_id}"
                 new_task = StoreTask(**task_options.dict())
+            elif task_id == "store_embedding":
```

Comment:
Store embedding is the action of saving an embedding or a set of embeddings to minio storage. Where vdb_upload (bulk ingestion) is so that all the written files can be ingested at once by milvus. While embed + vdb_upload (we can call it vanilla) is for streaming purposes, meaning that one embedding comes in and that same embedding is loaded in the milvus as they come through the pipe.